src/components/TodoItem.tsx/TodoItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react';
+import { TodoItemProp } from '../../types/Types';
+import classNames from 'classnames';
+import { UserIdContext, ActiveTodoContext } from '../../util/Store';
+
+export const TodoItem: React.FC<TodoItemProp> = ({ todoItem }) => {
+  const { setActiveUser } = useContext(UserIdContext);
+  const { todo, setTodo } = useContext(ActiveTodoContext);
+
+  return (
+    <tr data-cy="todo" className="">
+      <td className="is-vcentered">{todoItem.id}</td>
+      <td className="is-vcentered">
+        {todoItem.completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+      <td className="is-vcentered is-expanded">
+        <p
+          className={classNames(
+            todoItem.completed ? 'has-text-success' : 'has-text-danger',
```

Comment:
Use the classnames library rules for add classes with condition, fix it everywhere